trade-finance-manager-api/src/v1/rest-mappings/helpers/amendment.helpers.js

```diff
@@ -54,27 +61,29 @@ const findLatestCompletedAmendment = (amendments) => {
     }
     const existingUpdatedFields = cloneDeep(updatedFields);
 
-    if (!updatedFields.value) {
+    const hasFutureEffectiveDate = amendment.effectiveDate && isFutureEffectiveDate(amendment.effectiveDate);
```

Comment:
```suggestion
    const hasFutureEffectiveDate = isFutureEffectiveDate(amendment?.effectiveDate);
```

The `isFutureEffectiveDate` function should ascertain the `effectiveDate` property if null then should return falsy.